### PR TITLE
Implement changes for JWT recipe

### PR DESCRIPTION
**Pull request by nkshah2**
## Summary of change
This PR makes all the changes as defined in this PR (https://github.com/supertokens/supertokens-plugin-interface/pull/22) for the plugin-interface

## Related issues
- https://github.com/supertokens/for-zenhub/issues/16

## Related Pull Requests
- https://github.com/supertokens/supertokens-plugin-interface/pull/22
- https://github.com/supertokens/supertokens-core/pull/302
- https://github.com/supertokens/docs/pull/118

## Test Plan
All relevant tests have been added to supertokens-core (https://github.com/supertokens/supertokens-core/pull/302)

#### Core tests
All tests (existing + added in the above PR) pass

Screenshot 2021-09-10 at 4 43 57 PM

#### Plugin tests
All tests pass

Screenshot 2021-09-10 at 4 44 05 PM

## Documentation changes
- [x] Add `CREATE TABLE` query to database setup guides for recipes to include create query for the `jwt_signing_keys` table (https://github.com/supertokens/docs/pull/118)

## Checklist for important updates
- [x] Changelog has been updated
- [x] `pluginInterfaceSupported.json` file has been updated (if needed)
- [x] Changes to the version if needed
   - In `build.gradle`
- [x] Had installed and ran the pre-commit hook
- [x] ~~If there are new dependencies that have been added in `build.gradle`, please make sure to add them in~~ ~~`implementationDependencies.json`.~~
- [x] Issue this PR against the latest non released version branch.
   - To know which one it is, run find the latest released tag (`git tag`) in the format `vX.Y.Z`, and then find the latest branch (`git branch --all`) whose `X.Y` is greater than the latest released tag.
   - If no such branch exists, then create one from the latest released branch.

## Remaining TODOs for this PR
- [x] Changes to documentation website (https://github.com/supertokens/docs/pull/118)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [1.10.0] - 2021-0.9-10
```
Comment (rishabhpoddar) [inline CHANGELOG.md:10]: Typo in date

File: pluginInterfaceSupported.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "_comment": "contains a list of plugin interfaces branch names that this core supports",
   "versions": [
-    "2.8"
+    "2.8",
+    "2.9"
```
Comment (rishabhpoddar) [inline pluginInterfaceSupported.json:5]: Just 2.9

File: src/main/java/io/supertokens/storage/mysql/queries/JWTSigningQueries.java [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+                + "key_id VARCHAR(255) NOT NULL," + "key_string TEXT NOT NULL,"
+                + "algorithm VARCHAR(10) NOT NULL," + "created_at BIGINT UNSIGNED,"
+                + "PRIMARY KEY(key_id));";
+    }
+
+    public static List<JWTSigningKeyInfo> getJWTSigningKeys_Transaction(Start start, Connection con)
+            throws SQLException, StorageQueryException {
+        String QUERY = "SELECT * FROM "
+                + Config.getConfig(start).getJWTSigningKeysTable()
+                + " ORDER BY created_at DESC FOR UPDATE;";
```
Comment (rishabhpoddar) [inline src/main/java/io/supertokens/storage/mysql/queries/JWTSigningQueries.java:46]: Check if you need to add a ";" in the query.

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (nkshah2) [reply CHANGELOG.md]: Done

File: pluginInterfaceSupported.json (hunk 2 above)
Comment (nkshah2) [reply pluginInterfaceSupported.json:4]: Done

File: src/main/java/io/supertokens/storage/mysql/queries/JWTSigningQueries.java (hunk 3 above)
Comment (nkshah2) [reply src/main/java/io/supertokens/storage/mysql/queries/JWTSigningQueries.java]: It works without it, removed the semicolon

**Review by rishabhpoddar — APPROVED** [review]

**Pull request merged**